packages/dom-expressions/src/client.js

```diff
@@ -366,15 +364,16 @@ function assignProp(node, prop, value, prev, isSVG, skipRef, props) {
     (isChildProp = ChildProperties.has(prop)) ||
     (!isSVG &&
       ((propAlias = getPropAlias(prop, node.tagName)) || (isProp = Properties.has(prop)))) ||
-    (isCE = (node.nodeName.includes("-") || 'is' in props))
+    (isCE = node.nodeName.includes("-") || "is" in props)
   ) {
     if (forceProp) {
       prop = prop.slice(5);
       isProp = true;
     } else if (isHydrating(node)) return value;
     if (prop === "class" || prop === "className") className(node, value);
-    else if (isCE && !isProp && !isChildProp) node[toPropertyName(prop)] = value;
-    else node[propAlias || prop] = value;
+    else if (isCE && !isProp && !isChildProp) {
```

Comment:
Seems like we wouldn't need `isCE` anymore. The logic would be for any element (doesn't matter if it is CE or not) set the attribute (except for the special cases).

Reply:
Yeah that is the benefit here. Not treating CE's special. I think that is directionally where we go. I sort of over compensated because I wanted WC development to be pleasant but we shouldn't really be trying to pave over the awkward parts. They are the way they are and its up to the specs to handle that.